packages/offix-client/src/OfflineClient.ts

```diff
@@ -60,13 +61,12 @@ export class OfflineClient implements ListenerProvider {
   /**
   * Initialize client
   */
-  public async init(): Promise<ApolloOfflineClient> {
+  public async init(httpOptions?: HttpLink.Options): Promise<ApolloOfflineClient> {
```

Comment:
@darahayes @wtrocki If we are saying we are going to move to passing terminating links then I don't think we should break the api here. It would make more sense to add a `httpOptions` key to the config and use it from there. Otherwise, we will be making two destructive changes one after the other.